cmd/template/cluster/provider/templates/openstack/open_stack_machine_template.yaml.tmpl

```diff
@@ -0,0 +1,24 @@
+apiVersion: infrastructure.cluster.x-k8s.io/v1alpha4
+kind: OpenStackMachineTemplate
+metadata:
+  labels:
+    "release.giantswarm.io/version": "{{ .ReleaseVersion }}"
+    "giantswarm.io/cluster": "{{ .Name }}"
+    "cluster.x-k8s.io/cluster-name": "{{ .Name }}"
+    "giantswarm.io/organization": "{{ .Organization }}"
+  name: {{ .Name }}
+  namespace: {{ .Namespace }}
+spec:
+  template:
+    spec:
+      flavor: {{ .NodeMachineFlavor }}
```

Comment:
Is it `.NodeMachineFlavor` or `.ControlPlaneMachineFlavor` that needs to be set here?

Reply:
This template is shared for both machine_deployment.yaml.tmpl and kubeadm_control_plane.yaml.tmpl . Does it make sense to have a control plane machine template that is separate to workers?

There are a number of properties that I could see being different down the road.
https://github.com/kubernetes-sigs/cluster-api-provider-openstack/blob/main/config/crd/bases/infrastructure.cluster.x-k8s.io_openstackmachinetemplates.yaml#L73

Reply:
Good point. I want to keep this as simple as possible for now so I will remove `.ControlPlaneMachineFlavor` for now. When we need it we can add a second machine template for the control plane.